aspire_cloud_hosting_policy.md

```diff
@@ -0,0 +1,53 @@
+
+This DRAFT policy governs what software will be accepted to host on AspirePress Cloud. At the project's inception, we are providing a general list of requirements and deferring to the corresponding [guidelines available at wordpress.org](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/) for interpretation until they have been fully rewritten in this document. While this document is in draft form, in case of conflicts or updates, the guideline explanations below will supersede any others.
+
+If you believe you have a case where a software package would not be accepted for hosting on wordpress.org, but believe it should be allowed on Aspire Cloud, please contact support@aspirepress.org.
+
+While this Preamble should help understand and interpret the Hosting Policy, it is not a formal part of Policy.
+
+-----
+
+## Aspire Cloud Guidelines, Terms, & Conditions
+
+### Licensing & Other Requirements
+
+1. Plugins must be compatible with the GNU General Public License v2, but may use any license which qualifies.
+2. By submitting your software and updates, you agree to indemnify and hold harmless AspirePress and its network partners for any actions resulting from your software.
+3. Code must be (mostly) human readable.
+4. Developers are responsible for the contents and actions of their plugins.
+5. A stable version of a plugin must be available from its WordPress Plugin Directory page.
+6. Plugins must respect trademarks, copyrights, and project names.
+7. Developers and their plugins must not do anything illegal, dishonest, or morally offensive.
+8. A complete plugin must be available at the time of submission.
+
+### Features & Functions Not Permitted
+
+8. Trialware is not permitted.
+9. Plugins may not track users without their explicit consent, and must offer a means for the user to opt out.
+10. Plugins may not send executable code via third-party systems.
+
+### Best Practice Guidelines
+
+11. Plugins should not hijack the admin dashboard.
+12. Plugins must use WordPress’ default libraries.
+13. Frequent commits to a plugin should be avoided.
+14. Plugin version numbers must be incremented for each new release. Semantic versioning is encouraged but not required.
+
+### Additional Conditions & Permission
+
+15. We reserve the right to maintain the Plugin Directory to the best of our ability.
+16. Software as a Service is permitted.
+
+### Community Requirements
+
+17. Software contributors are expected to abide by the AspirePress Code of Conduct.
+18. Public facing pages on WordPress.org (readmes) must not spam.
+19. Plugins may not embed external links or credits on the public site without explicitly asking the user’s permission.
```

Comment:
Need to clarify "the public site" here.  I assume this means the site the plugin is installed on, but not including admin pages?  Such links might be the very purpose of the plugin, and the user permits it by installing it.  I sort of get the idea of what this is about, but an example would help.